client/state/account-recovery/reset/test/actions.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useNock from 'test/helpers/use-nock';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+import {
+	fetchResetOptions,
+	fetchResetOptionsSuccess,
+	fetchResetOptionsError,
+} from '../actions';
+
+import {
+	ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+} from 'state/action-types';
+
+describe( '#fetchResetOptionsSuccess', () => {
+	it( 'should return ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE action with options field.', () => {
+		const items = {
+			primaryEmail: 'primary@example.com',
+			primarySms: '12345678',
+			secondaryEmail: 'secondary@example.com',
+			secondarySms: '12345678',
+		};
+
+		const action = fetchResetOptionsSuccess( items );
+
+		assert.deepEqual( action, {
+			type: ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+			items,
+		} );
+	} );
+} );
+
+describe( '#fetchResetOptionsError', () => {
+	it( 'should return ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR action with error field.', () => {
+		const error = {
+			status: 400,
+			message: 'error!',
+		};
+
+		const action = fetchResetOptionsError( error );
+
+		assert.deepEqual( action, {
+			type: ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+			error,
+		} );
+	} );
+} );
+
+describe( '#fetchResetOptions', () => {
+	let spy;
+
+	useSandbox( sandbox => ( spy = sandbox.spy() ) );
+
+	const apiBaseUrl = 'https://public-api.wordpress.com:443';
+	const endpoint = '/wpcom/v2/account-recovery/lookup';
+
+	const userData = {
+		user: 'foo',
+	};
+
+	describe( 'success', () => {
+		const response = {
+			primary_email: 'a****@example.com',
+			secondary_email: 'b*****@example.com',
+			primary_sms: '+1******456',
+			secondary_sms: '+8*******456',
+		};
+
+		useNock( nock => (
+			nock( apiBaseUrl )
+				.get( endpoint )
+				.reply( 200, response )
+		) );
+
+		it( 'should dispatch RECEIVE action on success', () => {
+			const thunk = fetchResetOptions( userData )( spy );
+
+			assert.isTrue( spy.calledWith( {
+				type: ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+			} ) );
```

Comment:
Using `expect`, you can directly assert on the spies and get better error messages (instead of "expected false to be true"):

```js
expect( spy ).to.have.been.calledWith( { type: ... } );
```